### DISCUSSION TITLE
Further React compability

### DISCUSSION THREAD

**Discussion by ducan-ne** [honojs/hono#3356] [Ideas]
As the rise of https://github.com/honojs/hono/issues/2925
I think it'd be awesome to support some libraries that's common in react
For example React Aria Components is a lib I use the most and can cover almost use cases in my projects, I think it as the primitives for the web.

I currently get this error when trying use RAC
![CleanShot 2024-09-01 at 20 57 11@2x](https://github.com/user-attachments/assets/367d7978-8b53-4ce6-b533-6306d95e511c)

After I dig to see, it seems because hono/jsx event doesn't have event.nativeEvent property
![CleanShot 2024-09-01 at 20 58 35@2x](https://github.com/user-attachments/assets/2140221d-6e67-4759-9735-a563fbf4783a)

I know this is quite hard to support lib like Rac but just try to raise to see if someone is interested in.

**Comment by harel-tussi** [comment]
Could not agree more, I think supporting radix will also be great
